liquibase-core/src/test/groovy/liquibase/parser/core/xml/XMLChangeLogSAXParserTest.groovy

```diff
@@ -32,6 +32,21 @@ class XMLChangeLogSAXParserTest extends Specification {
         <output>&insecure;</output>
     </changeSet>
 
+</databaseChangeLog>
+"""
+
+    def INVALID_XML = """
+<!DOCTYPE databaseChangeLog [
+        <!ENTITY insecure SYSTEM "file:///invalid.txt">
+        ]>
+
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.19.xsd">
+
+    <iDontKnowWhatImDoing />
```

Comment:
Two things: 

1. The `<iDontKnowWhatImDoing/> made me laugh!
2. We should probably change the dbchangelog XSD version to be *-latest.xsd unless there is a reason to "pin" the version to 4.19.

Reply:
Agreed, changed

Reply:
Github is still showing me `http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.19.xsd">` ... Is it lying, lol?

Reply:
I guess it is 🤣

Reply:
So it was! I didn't see any of your changes, lol. We're all good now, though.